### PR TITLE
trigger-statements: Add general purpose statement

### PR DESCRIPTION
**Pull request by Udayan12167**

### CODE REVIEW

**Comment by sils** [comment]
Hey! This wasn't updated since a while. Please check up on it!

**Comment by sils** [comment]
@mr-karan can you check up on this? You're maintainer

**Comment by sils** [comment]
@Udayan12167 rebase?

File: trigger-statements.txt [hunk 1]
```diff
@@ -9,3 +9,16 @@ Users of multiple linters
...
+
+
+General purpose trigger statement
+---------------------------------
+
+coala provides a common multiplatform interface for linting and fixing all
+your code, regardless of the programming language you use. Its convenient user
+interface eliminates a lot of common tasks for the developer. coala also
+facilitates people entering the world of open source by providing them
+valuable feedback on coding standards and formatting. So, in a nutshell coala
```
Comment (adtac) [inline trigger-statements.txt]: comma after nutshell?

**Comment by adtac** [comment]
@Udayan12167 ping for rebase :)

File: trigger-statements.txt (hunk 1 above)
Comment (jayvdb) [inline trigger-statements.txt]: 'provides ... feedback..'
I think 'feedback' is the wrong word here, as it is a warm word, suggesting an intelligent agent.
coala is currently a cold heartless and very low intelligence feedback.

I think the intended point is coala simplifies the path for new OSS developers by giving them painless access to verification tools used by most OSS projects.

**Comment by jayvdb** [comment]
@Udayan12167 , bump.  this is quite good

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Comment by jayvdb** [comment]
Hey! I'm [GitMate.io](https://gitmate.io)! This pull request is being fastforwarded automatically. Please **DO NOT** push while fastforward is in progress or your changes would be lost permanently :warning:

**Comment by jayvdb** [comment]
Automated fastforward with [GitMate.io](https://gitmate.io) was successful! :tada:

**Pull request merged**